kits/infrastructure-risk-analyzer/agent.md

```
@@ -0,0 +1,16 @@
+# Infrastructure Risk Analyzer
+
+## Overview
```

markdownlint-cli2 (0.23.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)